tests/test_dictdiffer.py

```diff
@@ -195,6 +195,11 @@ def test_types(self):
         diffed = next(diff(first, second))
         assert ('change', 'a', (['a'], 'a')) == diffed
 
+    def test_nan(self):
+        value = float('nan')
+        diffed = list(diff([value], [value]))
+        assert diffed == []
+    
```

Comment:
ensuring decimal.Decimal(‘nan’) behaves like float(‘nan’) is up to the python language, not really the focus of this test.

> On Oct 5, 2015, at 12:59 AM, Jiri Kuncar <notifications@github.com> wrote:
> 
> In tests/test_dictdiffer.py <https://github.com/inveniosoftware/dictdiffer/pull/55#discussion_r41118083>:
> 
> > @@ -195,6 +195,11 @@ def test_types(self):
> >          diffed = next(diff(first, second))
> >          assert ('change', 'a', (['a'], 'a')) == diffed
> >  
> > +    def test_nan(self):
> > +        value = float('nan')
> > +        diffed = list(diff([value], [value]))
> > +        assert diffed == []
> > +    
> please test also float('nan') and number
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/inveniosoftware/dictdiffer/pull/55/files#r41118083>.
>